frontend/src/components/Banner.tsx

```diff
@@ -69,48 +52,16 @@ export const Banner = (props: BannerProps) => {
             {props.title}
           </h2>
         )) ||
-        (type === "info" && <h2 className={styles.title}>{props.title}</h2>)
+        (["info", "promo"].includes(type) && (
```

Comment:
Looks like we forgot to display the title for `promo` banners as well - this fixes that.